src/administration/integrations/blackfire.md

```diff
@@ -5,20 +5,25 @@ developed by [SensioLabs](https://sensiolabs.com/).
 
 ## Version
 
-* Agent/Client: `1.12.0`
-* Probe: `1.16.1` (when using `php:7.0` or `php:7.1`)
+* Agent/Client: `1.15.0`
+* Probe: `1.18.0` (when using `php:7.0` or `php:7.1`)
+* Chrome Companion: `1.15.0`
 
 ## Get Started
 
 ### 1. Get your credentials
 
-**Sign up for free** at [blackfire.io](https://blackfire.io/signup) and
+**Sign up for the free 15 days trial** at [blackfire.io](https://blackfire.io/signup) and
```

Comment:
This change looks wrong to me. You can sign up for a free account which lasts forever.